### PR TITLE
Update docs, rename to pivotal-lint [#169931786]

### PR DESCRIPTION
**Pull request by rheaditi**
* Updates the documentation putting usage first
* Adds more examples of comments added by `pivotal-lint`
* Renames the action to `pivotal-lint` wherever applicable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +42,61 @@ name: PR lint
...
+
+- A _team name_ label based on the PivotalTracker Project name (the project the story belongs to). For example, if your project name is `Escher POD` then it will add `escher` as a label.
+- `HOTFIX-PROD` - if the PR is raised against `production-release`.
+- `HOTFIX-PRE-PROD` - if the PR is raised against `release/v*`.
+- Pivotal story type (*feature*, *chore*, *bug*).
+
+<figure>
+ <img src="https://assets1.cleartax-cdn.com/cleargst-frontend/misc/1568800226_pr-lint.png" alt="Story details and labels added to a PR" />
+ <figcaption>
+ Story details and labels added to a PR [<a href="https://assets1.cleartax-cdn.com/cleargst-frontend/misc/1568800226_pr-lint.png">expand</a>].
```
Comment (rajanand02) [inline README.md]: @rheaditi this is not required

File: README.md [hunk 2]
```diff
@@ -52,22 +111,21 @@ Since tokens are private, we suggest adding them as [GitHub secrets](https://hel
...
 
 Example:
 
 ```yaml
 # ...
-  pr_lint:
     steps:
-    - name: Verify Pivotal story ID & add relevant labels
-      uses: cleartax/pr-lint@v2.1.0
+    - uses: cleartax/pivotal-lint@v2.1.0
```
Comment (rajanand02) [inline README.md]: Please add `pivotal-lint@latest`

File: README.md [hunk 3]
```diff
@@ -38,6 +42,61 @@ name: PR lint
...
+ </figcaption>
+</figure>
+
+#### Soft-validations via comments
+
+`pivotal-lint` will add comments to a PR to encourage better PR practices:
+
+**A good PR title**
+
+<figure>
```
Comment (rajanand02) [inline README.md]: incorrect image

File: README.md (hunk 2 above)
Comment (rheaditi) [reply README.md]: Sounds good! Done for the `@master` reference instead of this one!

File: README.md (hunk 3 above)
Comment (rheaditi) [reply README.md]: Fixed!

**Pull request merged**